### PR TITLE
Mobile issues

### PR DESCRIPTION
**Pull request by eliabejr**
Fixes for these issues:
https://github.com/Astrocoders/astrocoders.github.io/issues/11
https://github.com/Astrocoders/astrocoders.github.io/issues/14
https://github.com/Astrocoders/astrocoders.github.io/issues/15

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -70,6 +71,9 @@ const Footer = () => (
           <a href="https://twitter.com/_astrocoders" target="_blank" rel="noopener noreferrer">
             <Twitter />
           </a>
+          <a href="https://medium.com/astrocoders" target="_blank" rel="noopener noreferrer">
```
Comment (guilhermedecampo) [inline src/components/Footer.js:74]: Top

File: src/components/Footer.js [hunk 2]
```diff
@@ -70,6 +71,9 @@ const Footer = () => (
...
+            <Medium />
+          </a>
         </SocialLinks>
         <Copyrights>Copyright 2018 © Astrocoders Desenvolvimento LTDA</Copyrights>
```
Comment (guilhermedecampo) [inline src/components/Footer.js]: We got a feedback to write this here in English as well. 
So maybe just `Copyright {new Date().getFullYear()} © Astrocoders`

File: src/pages/case/cleanio.js [hunk 3]
```diff
@@ -14,7 +14,6 @@ const ProjectCover = styled.div`
   display: flex;
   flex-direction: column;
   justify-content: center;
-  margin-top: 126px;
```
Comment (guilhermedecampo) [inline src/pages/case/cleanio.js:17]: good catch

Comment (guilhermedecampo) [reply src/pages/case/cleanio.js:17]: since we are repeating these cases components let's extract to a single component to be imported on every case page?

Comment (eliabejr) [reply src/pages/case/cleanio.js:17]: can be!!! I used separated files thinking in different ways of expose what we do in the project, but, really, we aren't using this now

**Review by guilhermedecampo — APPROVED** [review]
Great!

**Comment by guilhermedecampo** [comment]
Meeerge 🚀

**Pull request merged**